JsonFile.js

```diff
@@ -205,6 +263,8 @@ Object.assign(JsonFile, {
   deleteKeyAsync,
   deleteKeysAsync,
   rewriteAsync,
-});
+};
+
+Object.assign(JsonFile, fns);
```

Comment:
Should both the class methods and the module methods use locking, or should there be an option to not do so?